### PR TITLE
Run xunit tests from build output directory

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: build/shade/_xunit-test.shade [hunk 1]
```diff
@@ -32,8 +32,7 @@ configuration=''
         }
 
         var testArgs = noParallelTestProjects.Contains(projectName) ? " -parallel none" : "";
-        var runnerFolder = Path.GetFullPath(Path.Combine(KoreBuildFolderPath, "xunit.runner.console", "2.1.0", "tools"));
-        var xunitCoreFolder = Path.GetFullPath(Path.Combine(KoreBuildFolderPath, "xunit.core", "2.1.0", "build", "_desktop"));
```
Comment (pranavkm) [inline build/shade/_xunit-test.shade:36]: This binary gets pulled in via `xunit.extensibility.desktop`

**Comment by pranavkm** [comment]
cc @victorhurdugaci \ @davidfowl

**Comment by victorhurdugaci** [comment]
cc @JunTaoLuo

**Comment by victorhurdugaci** [comment]
It looks okay assuming it doesn't break any tests that look for relative paths and it doesn't break the custom attribute discovery

**Comment by victorhurdugaci** [comment]
Oh, and that it works on mono too

**Comment by pranavkm** [comment]
We can address relative path test (Mvc and Localization are the only ones I know that use it). We can bring back specialized code if it doesn't work on Mono (I can't think of any reasons for it to fail?)

**Comment by victorhurdugaci** [comment]
If it doesn't work on mono, why do you want specialized code?

**Comment by pranavkm** [comment]
_if_ it doesn't work on Mono. I'm not sure it doesn't from the one repo I tried out.

**Comment by JunTaoLuo** [comment]
LGTM, thought I do recall seeing a few more places that use relative paths like Performance. I guess we'll just keep an eye on what fails.

**Comment by pranavkm** [comment]
Cool. It saves a bunch of time in Mvc because publish is never incremental.

**Comment by muratg** [comment]
:shipit: and fix as errors pop up :)

**Pull request closed without merging**